src/SFA.DAS.RoatpOversight/SFA.DAS.RoatpOversight.Web/Domain/Roles.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Security.Claims;
+
+namespace SFA.DAS.RoatpOversight.Web.Domain
+{
+    public static class Roles
+    {
+        public const string RoleClaimType = "http://service/service";
+
+        public const string RoatpGatewayTeam = "APR";
+        public const string RoatpGatewayAssessorTeam = "GAC";
+        public const string RoatpFinancialAssessorTeam = "FHC"; // RoATP FHA
+        public const string RoatpApplicationOversightTeam = "AOV";
+        public const string RoatpOversightTeam = "AAC";
+
+        public static bool HasValidRole(this ClaimsPrincipal user)
+        {
+            return user.IsInRole(RoatpGatewayTeam)
```

Comment:
Should this just be the AOV role?

Reply:
not sure at this point.  Certainly the link and subsequent pages should only be AOV, but not sure about the more general pages, such as 'Access Denied'.  I hope that makes sense.  I'll restrict it to AOV for now, and we'll figure it out from there.